SFML_TEST/SFML_TEST.cpp

```diff
@@ -0,0 +1,182 @@
+﻿#include <SFML/Window.hpp>
+#include <SFML/Graphics.hpp>
+#include <iostream>
+#include <algorithm>
+
+const float FIELD_WIDTH = 3440.0f;
+const float FIELD_HEIGHT = 1440.0f;
```

Comment:
Not the best idea. Won't work correctly with different WindowMode and especially with low-res monitors

Reply:
Better use size of the sf::RenderWindow object to always have actual window size which can be changed even at runtime